wrappers/readme.md

```diff
@@ -1,6 +1,6 @@
 # Official Wrappers
 
 * [Python](./python/) - supports both Python 2 and 3
-* Node.js - TBD
+* [Node.js](./nodejs) - support Node.js v6 above
```

Comment:
Great! We can also update the top-level ReadMe to encourage people to use your bindings

Reply:
toplevel readme is updated as well.

I'm going to add more commits to include linter (*.js, *.cpp) for CI build.

Reply:
BTW, I saw you adding back the appveyor.xml back, so I made the commit upon it. But the server side seems still using the different one (which only build python binding).